src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/DeactivateUacEndpoint.java

```diff
@@ -56,10 +55,10 @@ public void deactivateUac(
         uacQidLinkOpt.get().getCaze().getCollectionExercise().getSurvey(),
         CREATE_PRINT_TEMPLATE);
 
-    ResponseManagementEvent rme = new ResponseManagementEvent();
+    EventDTO rme = new EventDTO();
```

Comment:
This would be nicer as 'eventDto' rather than 'rme'

Reply:
fixed